tracee-ebpf/README.md

```diff
@@ -5,11 +5,11 @@
 [![License](https://img.shields.io/github/license/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/blob/main/LICENSE)
 [![docker](https://badgen.net/docker/pulls/aquasec/tracee)](https://hub.docker.com/r/aquasec/tracee)
 
-# Tracee - Container and system tracing using eBPF
+# Tracee-eBPF - Container and system tracing using eBPF
```

Comment:
Moving this readmen without creating a new one in the root directory will leave the repo without a readme

Reply:
I don't want to start debating the readme in this pr. Will add in future pr

Reply:
It's not about debating the readme - it's about having one for whoever enters the repo for the first time while there is no readme in the root dir

Reply:
I think that if we don't have another readme yet, we should keep this one until we will have.
Another option - open a readme which links to tracee-ebpf and tracee-rules readmes

Reply:
I will add something minimal then that points to tracee ebf and rules